README.md

```diff
@@ -1,3 +1 @@
-# Google Student Training in Engineering Program
-
-This repo contains Lily's portfolio and STEP projects.
+This repo contains Lily's portfolio and STEP projects.
```

Comment:
This includes the changes from your other PR (https://github.com/lilymzhou/stepfolio/pull/2).  Can you modify this PR to not include those changes?  I'm not sure if you specifically patched the changes into this branch or if creating a new branch brought the changes over.

In either case, the diff should only be showing new changes that are not present in your other PR.  Either this is a clean branch that doesn't show the other PR changes at all, or it does show them but they are not part of the diff because this change depends on that one to be submitted.

Reply:
My bad -- I accidentally created a branch off of LilyDevelopment rather than master. I just created a new pull request with just the most recent change (#6 ).

Reply:
Okay, no worries.  Thanks for creating the new pull request!